CHANGELOG.md

```diff
@@ -4,6 +4,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## hush-sensor [unreleased]
+
+### Added
+
+- add support for `sentry` `k8s` integration configuration
```

Comment:
Need to mention that it is enabled by default. So, if someone doesn't want it with the new chart need to disable explicitly. Possibly include a YAML snippet how to disable.